### PR TITLE
Bai 1419 implement the card from template endpoint on the on the python client

### PR DESCRIPTION
**Pull request by IR96334**

### CODE REVIEW

File: lib/python/docs/notebooks/models_and_releases_demo_pytorch.ipynb [hunk 1]
```diff
@@ -141,6 +141,38 @@
...
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model2 = Model.create(\n",
+    "    client=client, name=\"ResNet-50\", description=\"ResNet-50 model for image classification.\", team_id=\"uncategorised\"\n",
+    ")\n",
+    "\n",
+    "model2_id = model2.model_id\n",
+    "\n",
+    "model2.card_from_template(model2.model_id)\n",
```
Comment (GB27247) [inline lib/python/docs/notebooks/models_and_releases_demo_pytorch.ipynb:171]: I think this should use the model id from the previous model?

**Review by GB27247 — CHANGES_REQUESTED** [review]

File: lib/python/docs/notebooks/models_and_releases_demo_pytorch.ipynb (hunk 1 above)
Comment (IR96334) [reply lib/python/docs/notebooks/models_and_releases_demo_pytorch.ipynb:171]: good spot

File: lib/python/src/bailo/helper/entry.py [hunk 2]
```diff
@@ -62,12 +62,16 @@ def card_from_schema(self, schema_id: str | None = None) -> None:
 
         logger.info(f"Card for ID %s successfully created using schema ID %s.", self.id, schema_id)
 
-    def card_from_template(self):
+    def card_from_template(self, template_id: str) -> None:
         """Create a card using a template (not yet implemented).
 
-        :raises NotImplementedError: Not implemented error
+        #template is modelid? as it is not entry-specific, but model specific and you grab the template for the modelcard from
```
Comment (ARADDCC002) [inline lib/python/src/bailo/helper/entry.py:68]: Unneeded comment

File: lib/python/docs/notebooks/models_and_releases_demo_pytorch.ipynb [hunk 3]
```diff
@@ -141,6 +141,38 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Creating and populating a model card\n",
+    "\n",
+    "When creating a model card, first we need to generate an empty one using the `card_from_schema()` method. In this instance, we will use **minimal-general-v10**. You can manage custom schemas using the `Schema()` helper class, but this is out of scope for this demo.\n",
+    "\n",
+    "### Creating and populated a new model card with a template\n",
+    "\n",
+    "When creating a model card from a template, we need to use a preexisting model card as our template. First we create a new model, to create the new model card we use the `card_from_template` method and pass our chosen template model's id. \n",
```
Comment (ARADDCC002) [inline lib/python/docs/notebooks/models_and_releases_demo_pytorch.ipynb:154]: I would capitalise ID in this text

**Review by ARADDCC002 — CHANGES_REQUESTED** [review]

File: lib/python/src/bailo/helper/entry.py (hunk 2 above)
Comment (IR96334) [reply lib/python/src/bailo/helper/entry.py]: Done

File: lib/python/docs/notebooks/models_and_releases_demo_pytorch.ipynb (hunk 3 above)
Comment (IR96334) [reply lib/python/docs/notebooks/models_and_releases_demo_pytorch.ipynb]: Done

**Review by ARADDCC002 — APPROVED** [review]

**Review by ARADDCC012 — APPROVED** [review]

**Review by GB27247 — APPROVED** [review]

**Pull request merged**